### PR TITLE
Add Arabic (Saudi Arabia) translation

### PR DESCRIPTION
**Pull request by FiberAhmed**
**Sorry for the previous pull request but it was messed by mistake, sorry.**

### CODE REVIEW

**Comment by sylveon** [comment]
Can you sign the CLA please? Also, I've never tried TTB with a RTL language, I'm curious to see how it renders and if there any issues with RTL. Mind sending a few screenshots, maybe?

**Comment by FiberAhmed** [comment]
> Can you sign the CLA please? Also, I've never tried TTB with a RTL language, I'm curious to see how it renders and if there any issues with RTL. Mind sending a few screenshots, maybe?

I signed CLA

**Comment by FiberAhmed** [comment]
> Can you sign the CLA please? Also, I've never tried TTB with a RTL language, I'm curious to see how it renders and if there any issues with RTL. Mind sending a few screenshots, maybe?

Now what happened?

**Comment by Rexogamer** [comment]
They're wondering how the app looks with right-to-left text/would prefer to see a few screenshots

**Comment by sylveon** [comment]
Yeah, it seems handling RTL is a bit more involved: https://docs.microsoft.com/en-us/windows/apps/design/globalizing/adjust-layout-and-fonts--and-support-rtl#handling-right-to-left-rtl-languages

File: TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+/////////////////////////////////////////////////////////////////////////////
+//
+// Strings
+//
+
+STRINGTABLE
+{
+	IDS_WELCOME_NOTIFICATION APP_NAME L" يعيش في علبة النظام.\n\nانقر بزر الماوس الأيمن فوق رمز العلبة لتغيير الإعدادات، وتعيين التطبيق لفتحه عند التمهيد، وعرض المعلومات، والخروج من التطبيق، وما إلى ذلك."
```
Comment (ahmed605) [inline TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2:59]: I think this should look like this instead
```
IDS_WELCOME_NOTIFICATION L".يعيش في منطقة الإشعارات " APP_NAME L"\n\n.انقر بزر الماوس الأيمن فوق أيقونة الإشعار لتغيير الإعدادات، وتعيين التطبيق لفتحه عند بدء تشغيل النظام، وعرض المعلومات، والخروج من التطبيق، وما إلى ذلك"
```

File: TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2 [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	IDS_HIDETRAY_DIALOG L"هذا التغيير مؤقت وسيتم فقده في المرة التالية " APP_NAME L" التي يتم فيها البدء. \n\nلجعل هذا الأمر دائمًا، قم بتحرير ملف التكوين باستخدام ""خيارات متقدمة"" > ""تعديل الإعدادات"" واضبط ""hide_tray"" "على false.\n\nهل انت متأكد انك تريد المتابعة؟"
```
Comment (ahmed605) [inline TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2:60]: `false` should be `true` instead, see: https://github.com/TranslucentTB/TranslucentTB/blob/386ebf11eeeea0666981355474dd64cba70c8e2a/TranslucentTB/resources/language/TranslucentTB.en-US.rc2#L60

Suggested edits:
```
IDS_HIDETRAY_DIALOG APP_NAME L" هذا التغيير مؤقت وسيتم فقده في المرة التالية التي يتم فيها بدء. \n\n.true على hide_tray لجعل هذا الأمر دائمًا، قم بتحرير ملف التكوين باستخدام ""خيارات متقدمة"" > ""تعديل الإعدادات"" واضبط\n\nهل انت متأكد انك تريد المتابعة؟"
```

File: TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2 [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	IDS_ALREADY_RUNNING APP_NAME L" انه قيد التشغيل بالفعل! انقر بزر الماوس الأيمن على أيقونة العلبة للوصول إليها."
```
Comment (ahmed605) [inline TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2:61]: Suggested edits:
```
IDS_ALREADY_RUNNING L".قيد التشغيل بالفعل! انقر بزر الماوس الأيمن على أيقونة الإشعار للوصول إليه " APP_NAME
```

File: AppPackage/Strings/ar-SA/Resources.resw [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AppDescription" xml:space="preserve">
+    <value>أداة مساعدة خفيفة الوزن تجعل شريط مهام الويندوز شفافًا</value>
+  </data>
+  <data name="PublisherDisplayName" xml:space="preserve">
+    <value>مطوِّرو TranslucentTB مفتوح المصدر</value>
```
Comment (ahmed605) [inline AppPackage/Strings/ar-SA/Resources.resw:124]: Suggested edit:
```xml
    <value>مفتوح المصدر TranslucentTB مطوِّرو</value>
```

File: Xaml/Strings/ar-SA/Resources.resw [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ColorPickerPage_AlwaysOnTop.Text" xml:space="preserve">
+    <value>دائما في القمة</value>
```
Comment (ahmed605) [inline Xaml/Strings/ar-SA/Resources.resw:121]: Suggested edit:
```xml
    <value>دائما في المقدمة</value>
```

File: Xaml/Strings/ar-SA/Resources.resw [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+    <value>نعم</value>
+  </data>
+  <data name="ColorPickerPage_ConfirmCloseDialog.SecondaryButtonText" xml:space="preserve">
+    <value>لا</value>
+  </data>
+  <data name="FramelessPage_Close.Text" xml:space="preserve">
+    <value>إغلاق</value>
+  </data>
+  <data name="TrayFlyoutPage_About.Text" xml:space="preserve">
+    <value>حول TranslucentTB</value>
```
Comment (ahmed605) [inline Xaml/Strings/ar-SA/Resources.resw:139]: Suggested edit:
```xml
    <value>TranslucentTB حول</value>
```

File: Xaml/Strings/ar-SA/Resources.resw [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+    <value>تعطيل حفظ الإعدادات</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_DumpDynamicState.Text" xml:space="preserve">
+    <value>تفريغ الحالة الديناميكية لِلسجل</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_EditSettings.Text" xml:space="preserve">
+    <value>تعديل الإعدادات</value>
+  </data>
+  <data name="TrayFlyoutPage_Advanced_HideTrayIcon.Text" xml:space="preserve">
+    <value>إخفاء أيقونة العلبة</value>
```
Comment (ahmed605) [inline Xaml/Strings/ar-SA/Resources.resw:163]: Suggested edit:
```xml
    <value>إخفاء أيقونة الإشعار</value>
```

File: Xaml/Strings/ar-SA/Resources.resw [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+  </data>
+  <data name="TrayFlyoutPage_VisibleWindow.Text" xml:space="preserve">
+    <value>نافذة مرئية</value>
+  </data>
+  <data name="WelcomePage.Title" xml:space="preserve">
+    <value>مرحبًا بك في TranslucentTB!</value>
+  </data>
+  <data name="WelcomePage_Action_Configure.Description" xml:space="preserve">
+    <value>يمكنك دائمًا القيام بذلك عن طريق النقر فوق "خيارات متقدمة" &gt; "تحرير الإعدادات" في قائمة سياق رمز العلبة
+يتم إعادة تحميل التغييرات تلقائيًا عند حفظ الملف</value>
```
Comment (ahmed605) [inline Xaml/Strings/ar-SA/Resources.resw:251]: Suggested edit:
```xml
    <value>تحرير الإعدادات" في قائمة سياق رمز الإشعار" &gt; "يمكنك دائمًا القيام بذلك عن طريق النقر فوق "خيارات متقدمة
يتم إعادة تحميل التغييرات تلقائيًا عند حفظ الملف</value>
```

File: Xaml/Strings/ar-SA/Resources.resw [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+    <value>تحدث مع المطورين واحصل على مساعدة في البرنامج</value>
+  </data>
+  <data name="WelcomePage_Action_Discord.Name" xml:space="preserve">
+    <value>انضم إلى خادم الديسكورد الخاص بنا</value>
+  </data>
+  <data name="WelcomePage_Action_Donate.Description" xml:space="preserve">
+    <value>أظهر لنا دعمك من خلال تقديم تبرع لمرة واحدة أو مُتكرر ❤</value>
+  </data>
+  <data name="WelcomePage_Action_Donate.Name" xml:space="preserve">
+    <value>تبرع لمشروع ليبرا باي</value>
```
Comment (ahmed605) [inline Xaml/Strings/ar-SA/Resources.resw:267]: Suggested edit:
```xml
    <value>تبرع للمشروع على ليبرا باي</value>
```

File: Xaml/Strings/ar-SA/Resources.resw [hunk 10]
```diff
@@ -0,0 +1,285 @@
...
+  </data>
+  <data name="WelcomePage_ButtonBar_Continue.Content" xml:space="preserve">
+    <value>مُتابعة</value>
+  </data>
+  <data name="WelcomePage_ButtonBar_Exit.Content" xml:space="preserve">
+    <value>خروج</value>
+  </data>
+  <data name="WelcomePage_Message_Hyperlink.Text" xml:space="preserve">
+    <value>إفتح مُشكلة GitHub</value>
```
Comment (ahmed605) [inline Xaml/Strings/ar-SA/Resources.resw:276]: Suggested edit:
```xml
    <value>GitHub فتح طلب على</value>
```

File: Xaml/Strings/ar-SA/Resources.resw [hunk 11]
```diff
@@ -0,0 +1,285 @@
...
+  </data>
+  <data name="WelcomePage_Message_Prefix.Text" xml:space="preserve">
+    <value>شكرًا لِتجربة TranslucentTB! بعد النقر على "مُتابعة"، يمكنك تخصيص شكل شريط المهام وفقًا لرغباتك. نأمل أن تستمتع بها، ولا تتردد في ذلك</value>
```
Comment (ahmed605) [inline Xaml/Strings/ar-SA/Resources.resw:279]: Suggested edit:
```xml
    <value>بعد النقر على "مُتابعة"، يمكنك تخصيص شكل شريط المهام وفقًا لرغباتك. نأمل أن تستمتع بهذا، ولا تتردد في !TranslucentTB شكرًا لِتجربة</value>
```

**Review by ahmed605 — CHANGES_REQUESTED** [review]

File: AppPackage/Strings/ar-SA/Resources.resw (hunk 4 above)
Comment (littlewhitecloud) [reply AppPackage/Strings/ar-SA/Resources.resw:124]: @ahmed605 you can write "```suggestion```" to make a suggestion sush as:
image

```suggestion
    <value>مفتوح المصدر TranslucentTB مطوِّرو</value>
```

File: TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2 (hunk 3 above)
Comment (littlewhitecloud) [reply TranslucentTB/resources/language/TranslucentTB.ar-SA.rc2:61]:
```suggestion
	IDS_ALREADY_RUNNING L".قيد التشغيل بالفعل! انقر بزر الماوس الأيمن على أيقونة الإشعار للوصول إليه " APP_NAME
```

File: Xaml/Strings/ar-SA/Resources.resw (hunk 5 above)
Comment (littlewhitecloud) [reply Xaml/Strings/ar-SA/Resources.resw:121]:
```suggestion
    <value>دائما في المقدمة</value>
```

File: Xaml/Strings/ar-SA/Resources.resw (hunk 6 above)
Comment (littlewhitecloud) [reply Xaml/Strings/ar-SA/Resources.resw:139]:
```suggestion
    <value>TranslucentTB حول</value>
```

File: Xaml/Strings/ar-SA/Resources.resw (hunk 7 above)
Comment (littlewhitecloud) [reply Xaml/Strings/ar-SA/Resources.resw:163]:
```suggestion
    <value>إخفاء أيقونة الإشعار</value>
```

File: Xaml/Strings/ar-SA/Resources.resw (hunk 8 above)
Comment (littlewhitecloud) [reply Xaml/Strings/ar-SA/Resources.resw:251]:
```suggestion
    <value>تحرير الإعدادات" في قائمة سياق رمز الإشعار" &gt; "يمكنك دائمًا القيام بذلك عن طريق النقر فوق "خيارات متقدمة
يتم إعادة تحميل التغييرات تلقائيًا عند حفظ الملف</value>
```

File: Xaml/Strings/ar-SA/Resources.resw (hunk 9 above)
Comment (littlewhitecloud) [reply Xaml/Strings/ar-SA/Resources.resw:267]:
```suggestion
    <value>تبرع للمشروع على ليبرا باي</value>
```

File: Xaml/Strings/ar-SA/Resources.resw (hunk 10 above)
Comment (littlewhitecloud) [reply Xaml/Strings/ar-SA/Resources.resw:276]:
```suggestion
    <value>GitHub فتح طلب على</value>
```

File: Xaml/Strings/ar-SA/Resources.resw (hunk 11 above)
Comment (littlewhitecloud) [reply Xaml/Strings/ar-SA/Resources.resw:279]:
```suggestion
    <value>بعد النقر على "مُتابعة"، يمكنك تخصيص شكل شريط المهام وفقًا لرغباتك. نأمل أن تستمتع بهذا، ولا تتردد في !TranslucentTB شكرًا لِتجربة</value>
```

**Review by littlewhitecloud — COMMENTED** [review]
.

**Comment by sylveon** [comment]
Closing as no author reply

**Pull request closed without merging**